Update tutorials, closes MISC-212 (#61)
gatling/frontline-cloud-doc: content/tutorials/sbt/index.md

---
title: "Getting started with sbt"
menutitle: "sbt"
description: "Learn how to setup Gatling Enterprise using sbt"
date: 2021-03-19T17:03:31+01:00
lastmod: 2021-03-19T17:03:31+01:00
weight: 50
---

{{< include introduction.md >}}

## Download the sample Gatling simulation

There are multiple ways to create a Gatling artifact, which will be used by Gatling Enterprise to launch a run.

> You can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.

Reviewer: ```suggestion
You can use either a build tool (Maven, SBT, Gradle) or the Gatling bundle.
```
Notes:
- On the official SBT website, it's spelled "sbt" (all lowercase), but I think in most of the docs we use "SBT" (all uppercase). Let's at least be consistent.
- Same thing for the other tutorials.

In this tutorial, we'll use the **SBT** build tool, so make sure you have SBT configured.

> You'll use SBT from the terminal, but you can also do it easily with an IDE like IntelliJ.

Reviewer: ```suggestion
In this tutorial, we will use SBT from the terminal, but you can also easily run SBT commands from an IDE such as IntelliJ IDEA.
```

{{< include download_samples.md >}}

Extract the archive, then navigate to the **sbt** folder.

> This folder contains a basic Gatling simulation which can be used with FrontLine.

Reviewer: ```suggestion
This folder contains a basic Gatling simulation which can be used with Gatling Enterprise.
```

sbt will download the necessary dependencies and package your simulation.
That's it, you've created your first Gatling artifact!

## Upload the artifact 

{{< include upload_manual.md "frontline-samples/sbt/target/sbt-frontline-1.0.0.jar" >}}

Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.

{{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}

> You now have to click on the Upload button to upload it.
> After a few seconds, the upload will be complete, and your artifact will be ready to use!

Reviewer: ```suggestion
Click on the Upload button. After a few seconds, the upload will complete, and your artifact will be ready to use
```
